packages/libs/coreui/src/components/inputs/SelectList.tsx

```diff
@@ -1,12 +1,15 @@
 import { ReactNode, useEffect, useState } from 'react';
 import PopoverButton from '../buttons/PopoverButton/PopoverButton';
 import CheckboxList, { CheckboxListProps } from './checkboxes/CheckboxList';
+import Spinner from '@veupathdb/components/lib/components/Spinner';
```

Comment:
This breaks the build, because it causes a circular dependency. You might need to move the `Spinner` component to this repo. Or, just use text, like **Loading...**?

Reply:
"Loading..." is way better 👍